crates/evm/networks/README.md

```diff
@@ -0,0 +1,58 @@
+# Custom EVM Networks
```

Comment:
can we also add a note on this in the dev docs directory?

Reply:
pls check https://github.com/foundry-rs/foundry/pull/11943